aequilibrae/project/spatialite_connection.py

```diff
@@ -0,0 +1,21 @@
+import os
+import platform
+from aequilibrae.parameters import Parameters
+from aequilibrae import logger
+
+
+def spatialite_connection(conn):
+    conn.enable_load_extension(True)
+    plat = platform.platform()
+    pth = os.getcwd()
+    if "WINDOWS" in plat.upper():
+        par = Parameters()
+        spatialite_path = par.parameters["system"]["spatialite_path"]
+        if os.path.isfile(os.path.join(spatialite_path, "mod_spatialite.dll")):
+            os.chdir(spatialite_path)
```

Comment:
here you can just add that dir to the path 

![image](https://user-images.githubusercontent.com/151124/75393711-64108180-593a-11ea-859c-d5b5fda99263.png)

Reply:
It did not work on windows

Reply:
note the difference in the sys.path vs os.environ["Path"]. sys.path isn't enough

Reply:
Hummm...  Good point

Reply:
And it worked!  Will add this to a future PR